kansha/board/comp.py

```diff
@@ -62,10 +62,10 @@ class Board(object):
     max_shown_members = 4
     background_max_size = 3 * 1024  # in Bytes
 
-    def __init__(self, id_, app_title, app_banner, theme, search_engine,
+    def __init__(self, id_, app_title, app_banner, theme, card_extensions, search_engine,
                  assets_manager_service, mail_sender_service, services_service,
                  on_board_delete=None, on_board_archive=None,
-                 on_board_restore=None, on_board_leave=None, on_update_members=None,load_data=True):
+                 on_board_restore=None, on_board_leave=None, on_update_members=None, load_data=True):
```

Comment:
Note: one of the goals of the events system is to removed all these `on_*` callbacks